modules/virt-defining-storageclass-in-cdi.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-preparing-cdi-scratch-space.adoc
+
+[id="virt-defining-storageclass-in-cdi_{context}"]
+= Defining a storage class in the CDI object
+
+Define a storage class in the `CDI` object  to dynamically provision scratch space for CDI operations.
+
+.Procedure
+
+* Use the `oc` client to edit the `CDI` object and edit the value of the `spec.config.scratchSpaceStorageClass` field to match a storage class in the cluster.
```

Comment:
Should this module have a prereq for them to have the `oc` client?

Reply:
I've had folks question whether we really need to tell people this but I don't including it for completeness. It is necessary to complete the procedure after all :)